UOP1_Project/Assets/Scripts/Characters/Character.cs

```diff
@@ -4,16 +4,16 @@ public class Character : MonoBehaviour
 {
 	private CharacterController characterController;
 
-	[Tooltip("Horizontal XZ plane speed multiplier")] public float speed = 8f;
-	[Tooltip("Smoothing for rotating the character to their movement direction")] public float turnSmoothTime = 0.2f;
-	[Tooltip("General multiplier for gravity (affects jump and freefall)")] public float gravityMultiplier = 5f;
-	[Tooltip("The initial upwards push when pressing jump. This is injected into verticalMovement, and gradually cancelled by gravity")] public float initialJumpForce = 10f;
-	[Tooltip("How long can the player hold the jump button")] public float jumpInputDuration = .4f;
-	[Tooltip("Represents how fast gravityContributionMultiplier will go back to 1f. The higher, the faster")] public float gravityComebackMultiplier = 15f;
-	[Tooltip("The maximum speed reached when falling (in units/frame)")] public float maxFallSpeed = 50f;
-	[Tooltip("Each frame while jumping, gravity will be multiplied by this amount in an attempt to 'cancel it' (= jump higher)")] public float gravityDivider = .6f;
-	[Tooltip("Adjust the friction of the slope")] public float slideFriction = 0.3f;
-	[Tooltip("Starting vertical movement when falling from a platform")] public float fallingVerticalMovement = -5f;
+	[Tooltip("Horizontal XZ plane speed multiplier")] [SerializeField] private float speed = 8f;
+	[Tooltip("Smoothing for rotating the character to their movement direction")] [SerializeField] private float turnSmoothTime = 0.2f;
+	[Tooltip("General multiplier for gravity (affects jump and freefall)")] [SerializeField] private float gravityMultiplier = 5f;
+	[Tooltip("The initial upwards push when pressing jump. This is injected into verticalMovement, and gradually cancelled by gravity")] [SerializeField] private float initialJumpForce = 10f;
+	[Tooltip("How long can the player hold the jump button")] [SerializeField] private float jumpInputDuration = .4f;
+	[Tooltip("Represents how fast gravityContributionMultiplier will go back to 1f. The higher, the faster")] [SerializeField] private float gravityComebackMultiplier = 15f;
+	[Tooltip("The maximum speed reached when falling (in units/frame)")] [SerializeField] private float maxFallSpeed = 50f;
+	[Tooltip("Each frame while jumping, gravity will be multiplied by this amount in an attempt to 'cancel it' (= jump higher)")] [SerializeField] private float gravityDivider = .6f;
+	[Tooltip("Adjust the friction of the slope")] [SerializeField] private float slideFriction = 0.3f;
+	[Tooltip("Starting vertical movement when falling from a platform")] [SerializeField] private float fallingVerticalMovement = -5f;
```

Comment:
The naming convention for private fields is _camelCase.

Reply:
I'm surprised the linter hasn't caught that. Maybe because the CLA hasn't been signed yet?

Reply:
I think that must be it. Happened in #100 as well. Someone who hadn't signed the CLA had their non-compliant code merged into the PR. The code was manually made compliant in a commit, the CLA was signed, and the linter never ran.

I'd be curious to see if it would run automatically once the CLA is signed.

Reply:
@davejrodriguez Yes, I realize that I missed the _camelCase after I submitted the PR. I was so focused on getting all those public fields private and serialized that I missed the _camelCase. 

I am actually having trouble signing the CLA. Where should I report that?

Reply:
> I am actually having trouble signing the CLA. Where should I report that?

Not sure how to help with that, it's not an extension we made. Do you have more than one Github username? Sometimes that can be the issue.

Reply:
I closed this due to all the issues I was having. I had to fix my account with GitHub and then was able to sign. In doing so, I had to resubmit this after it was fixed.

Reply:
> > I am actually having trouble signing the CLA. Where should I report that?
> 
> Not sure how to help with that, it's not an extension we made. Do you have more than one Github username? Sometimes that can be the issue.

I closed this due to all the issues I was having. I had to fix my account with GitHub and then was able to sign. In doing so, I had to resubmit this after it was fixed.